spec/Schema.spec.js

```diff
@@ -309,6 +309,12 @@ describe('SchemaController', () => {
             foo: { type: 'String' },
           },
           classLevelPermissions: {
+            ACL: {
```

Comment:
```suggestion
            acl: {
```

Since we're using camelCase also for abbreviations, I'd suggest to rename this to `acl`. But if we are using `ACL` already in other places, we should probably keep it as `ACL` to avoid misconfig and at some point in the future change them all together.

Reply:
Yes the object level acl is written as `ACL`